# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse_capi/tests/test_workaround_stsc.rs
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 third_party/rust/mp4parse_capi/tests/test_workaround_stsc.rs
--- a/third_party/rust/mp4parse_capi/tests/test_workaround_stsc.rs	Fri Jan 27 19:06:16 2023 +0000
+++ b/third_party/rust/mp4parse_capi/tests/test_workaround_stsc.rs	Fri Jan 27 21:11:42 2023 +0200
@@ -1,3 +1,4 @@
+extern crate mp4parse_capi;
 use mp4parse_capi::*;
 use std::io::Read;
 